From b8fd038b6fa2fbf95e31ef1474e33335a4827eee Mon Sep 17 00:00:00 2001
From: xzl <xiangzelong@deepin.org>
Date: Thu, 30 May 2024 15:13:02 +0800
Subject: [PATCH 1/1] Disable synaptics and focaltech_moc

---
 tests/meson.build | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/meson.build b/tests/meson.build
index dc3b70e..44e3bb8 100644
--- a/tests/meson.build
+++ b/tests/meson.build
@@ -39,7 +39,7 @@ drivers_tests = [
     'elan-cobo',
     'elanmoc',
     'elanspi',
-    'synaptics',
+#    'synaptics',
     'upektc_img',
     'upektc_img-tcs1s',
     'uru4000-msv2',
@@ -55,7 +55,7 @@ drivers_tests = [
     'egismoc-05a1',
     'fpcmoc',
     'realtek',
-    'focaltech_moc',
+#    'focaltech_moc',
 ]
 
 if get_option('introspection')
-- 
2.43.4

